tools/LogCollector/source/omslinux_agentlog.py

```diff
@@ -401,6 +401,12 @@ def copyCommonFiles(omsLinuxType):
     cmd='cp /var/opt/microsoft/omsconfig/omsconfig* /tmp/omslogs'
     out=execCommand(cmd)
     writeLogCommand(cmd)
+    cmd='mkdir -p /tmp/omslogs/configuration'
```

Comment:
Please rename this folder, or copy your logs into another subfolder with a name indicating that it is just DSC config.

Reply:
changing folder name to dscconfiguration